server/migrations/20180927153526-create-user.js

```diff
@@ -9,6 +10,7 @@ module.exports = {
     },
     username: {
       allowNull: false,
+      primaryKey: true,
```

Comment:
`Profile model` references `User model` via username. I therefore make `User model's username field` a  primary key so that Sequelize association between the two models can work effectively.